pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -767,6 +768,89 @@ type BackupStorageS3Spec struct {
 	CredentialsSecret string `json:"credentialsSecret"`
 	Region            string `json:"region,omitempty"`
 	EndpointURL       string `json:"endpointUrl,omitempty"`
+
+	// CABundle specifies the CA bundle to use for the S3 connection.
+	// This is useful for self-signed certificates or certificates signed using custom CAs.
+	// This can be specified as a string or a Secret key selector.
+	// For example:
+	// 	caBundle: <YOUR BASE64 ENCODED CA BUNDLE STRING>
+	// or
+	// 	caBundle:
+	// 	  fromSecret:
+	// 	    name: <YOUR CA SECRET NAME>
+	// 		key: <KEY IN SECRET CONTAINING THE BASE64 ENCODED CA BUNDLE>
+	// +kubebuilder:validation:Schemaless
+	// +kubebuilder:pruning:PreserveUnknownFields
+	CABundle *StringOrDataSource `json:"caBundle,omitempty"`
```

Comment:
Wonder why we need to support both options and not just have the secret configuration? we do something similar for all our certificates as @hors suggested.

Reply:
> will the inline base64 encoded string be handled by cert manager automatically out of the box?

no, I don't think so.. cert-manager recognises only built-in APIs. I'm not aware of a way to extend it to CRs..

Reply:
Okay, in that case, we can remove the string and keep only the Secret reference 👍🏻